src/viam/ur/module/ur_arm.cpp

```diff
@@ -612,8 +625,9 @@ void URArm::move_to_joint_positions(const std::vector<double>& positions, const
     std::list<Eigen::VectorXd> waypoints;
     waypoints.emplace_back(std::move(next_waypoint_rad));
 
-    const auto unix_time = unix_now_ms();
+    const auto unix_time = unix_time_iso8601();
     const auto filename = waypoints_filename(current_state_->output_csv_dir_path, unix_time);
+
```

Comment:
⬇️ - Should `std::move` `unix_time` at call to `move_`. Alternatively, `move_` could take it by `const&` since it only observes it.

Reply:
made move_ take the const&